file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
+                puppet.bat *                  # executes Puppet and provides results of Puppet runs
```

Comment:
@ferventcoder this script is some ruby glue that allows `pxp-agent` to ask questions of puppet: https://github.com/puppetlabs/pxp-module-puppet/blob/master/modules/puppet

And yeah @parisiale, I don't know why Michael had a `.bat` wrapper. Do we *need* a particular file extension in order to be able to execute something?

Anyway, this may be more minutia than we need to definitively sort out in *this* PR. I.e. if we sort out the directory structure now, and later change our mind to make this not be a `.bat` file, we can just come back and update this.